content/_data/upgrades/2-221-1.adoc

```diff
@@ -0,0 +1,55 @@
+==== Header and Breadcrumb Layout Improvements
+
+Instances with plugins that depend on specific details of the Jenkins layout (like the https://plugins.jenkins.io/simple-theme-plugin/[Simple Theme plugin]) may have user interface issues with this release.
+
+==== Security Defaults
+
+Global security settings no longer have a user interface setting that will disable them.
+Previously, administrators could disable security with a single checkbox.
+They could also disable cross-site request forgery (CSRF) protection with a single checkbox.
+Those two settings **cannot be disabled** in this release.
+
+Outdated agent protocols that had previously been deprecated have been removed in this release.
+`Inbound TCP Agent Protocols` versions 1, 2, and 3 have been removed.
+The `agent.jar` and `slave.jar` files on agents must be updated to the `agent.jar` version shipped with this release.
+The latest `agent.jar` can be downloaded from \https://yourserver.example.com/jnlpJars/agent.jar .
+
+Password auto-fill is only allowed on the login password field.
+Password fields which are **not** the login password field will no longer allow password auto-fill.
+This avoids browser prompts to automatically fill password fields that are not the login password field.
+Password auto-fill for other password fields can be enabled with the system property:
+
+[source,java]
+----
+hudson.Functions.hidingPasswordFields=false
+----
+
+==== Security Deprecations
+
+The `Overall/RunScripts`, `Overall/UploadPlugins`, and `Overall/ConfigureUpdateCenter` permissions were disabled by default in 2017.
+Those permissions are now **deprecated**.
+Custom authorization strategy plugins that that grant `Overall/Administer` without implying oneor more of those permissions will no longer work as expected.
+
+==== Experimental User Interface Changes
+
+The experimental Jenkins user interface refresh can be enabled by setting the property `jenkins.ui.refresh=true`.
+The user interface refresh enables a new header color scheme.
+Additional user interface changes will be enabled in future releases with the `jenkins.ui.refresh=true` property.
```

Comment:
This entry has low value without a link to a full story and to ways to provide feedback